### ISSUE TITLE
MAILER_SENDER_EMAIL unexpected character "<" in variable name near "<accounts@chatwoot.com>

### ISSUE THREAD

**Issue by lybrus** [chatwoot/chatwoot#3217]
ENV variable MAILER_SENDER_EMAIL 

**Expectation**
> The email from which all outgoing emails are sent
> could user either  `email@yourdomain.com` or `BrandName <email@yourdomain.com>`

**Reality**

MAILER_SENDER_EMAIL=`BrandName <email@yourdomain.com>`
Rails doesn't start with `unexpected character "<" in variable name near "<accounts@chatwoot.com>`

How to reproduce

I just tried to run chatwoot in docker using instruction https://www.chatwoot.com/docs/self-hosted/deployment/docker Didn't change any default ENV variables values. On 4 step:
```shell
➜  chatwoot docker-compose run --rm rails bundle exec rails db:chatwoot_prepare
unexpected character "<" in variable name near "<accounts@chatwoot.com>\n\n\n#SMTP domain key is set up for HELO checking\nSMTP_DOMAIN=chatwoot.com\n# the default value is set \"mailhog\" and is used by docker-compose for development environments,\n# Set the value as \"localhost\" or your SMTP address in other environments\nSMTP_ADDRESS=mailhog\nSMTP_PORT=1025\nSMTP_USERNAME=\nSMTP_PASSWORD=\n# plain,login,cram_md5\nSMTP_AUTHENTICATION=\nSMTP_ENABLE_STARTTLS_AUTO=true\n# Can be: 'none', 'peer', 'client_once', 'fail_if_no_peer_cert', see http://api.rubyonrails.org/classes/ActionMailer/Base.html\nSMTP_OPENSSL_VERIFY_MODE=peer\n\n# Mail Incoming\n# This is the domain set for the reply emails when conversation continuity is enabled\nMAILER_INBOUND_EMAIL_DOMAIN=\n# Set this to appropriate ingress channel with regards to incoming emails\n# Possible values are :\n# relay for Exim, Postfix, Qmail\n# mailgun for Mailgun\n# mandrill for Mandrill\n# postmark for Postmark\n# sendgrid for Sendgrid\nRAILS_INBOUND_EMAIL_SERVICE=\n# Use one of the following based on the email ingress service\n# Ref: https://edgeguides.rubyonrails.org/action_mailbox_basics.html\nRAILS_INBOUND_EMAIL_PASSWORD=\nMAILGUN_INGRESS_SIGNING_KEY=\nMANDRILL_INGRESS_API_KEY=\n\n# Storage\nACTIVE_STORAGE_SERVICE=local\n\n# Amazon S3\n# documentation: https://www.chatwoot.com/docs/configuring-s3-bucket-as-cloud-storage\nS3_BUCKET_NAME=\nAWS_ACCESS_KEY_ID=\nAWS_SECRET_ACCESS_KEY=\nAWS_REGION=\n\n\n# Log settings\n# Disable if you want to write logs to a file\nRAILS_LOG_TO_STDOUT=true\nLOG_LEVEL=info\nLOG_SIZE=500\n\n### This environment variables are only required if you are setting up social media channels\n\n# Facebook\n# documentation: https://www.chatwoot.com/docs/facebook-setup\nFB_VERIFY_TOKEN=\nFB_APP_SECRET=\nFB_APP_ID=\n\n# https://developers.facebook.com/docs/messenger-platform/instagram/get-started#app-dashboard\nIG_VERIFY_TOKEN=\n\n# Twitter\n# documentation: https://www.chatwoot.com/docs/twitter-app-setup\nTWITTER_APP_ID=\nTWITTER_CONSUMER_KEY=\nTWITTER_CONSUMER_SECRET=\nTWITTER_ENVIRONMENT=\n\n#slack integration\nSLACK_CLIENT_ID=\nSLACK_CLIENT_SECRET=\n\n### Change this env variable only if you are using a custom build mobile app\n## Mobile app env variables\nIOS_APP_ID=6C953F3RX2.com.chatwoot.app\nANDROID_BUNDLE_ID=com.chatwoot.app\n\n# https://developers.google.com/android/guides/client-auth (use keytool to print the fingerprint in the first section)\nANDROID_SHA256_CERT_FINGERPRINT=AC:73:8E:DE:EB:56:EA:CC:10:87:02:A7:65:37:7B:38:D4:5D:D4:53:F8:3B:FB:D3:C6:28:64:1D:AA:08:1E:D8\n\n\n### Smart App Banner\n# https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/PromotingAppswithAppBanners/PromotingAppswithAppBanners.html\n# You can find your app-id in https://itunesconnect.apple.com\n#IOS_APP_IDENTIFIER=1495796682\n\n## Push Notification\n## generate a new key value here : https://d3v.one/vapid-key-generator/\n# VAPID_PUBLIC_KEY=\n# VAPID_PRIVATE_KEY=\n#\n# for mobile apps\n# FCM_SERVER_KEY=\n\n## Bot Customizations\nUSE_INBOX_AVATAR_FOR_BOT=true\n\n\n### APM and Error Monitoring configurations\n## Sentry\n# SENTRY_DSN=\n\n## Scout\n## https://scoutapm.com/docs/ruby/configuration\n# SCOUT_KEY=YOURKEY\n# SCOUT_NAME=YOURAPPNAME (Production)\n# SCOUT_MONITOR=true\n\n## NewRelic\n# https://docs.newrelic.com/docs/agents/ruby-agent/configuration/ruby-agent-configuration/\n# NEW_RELIC_LICENSE_KEY=\n\n## Datadog\n## https://github.com/DataDog/dd-trace-rb/blob/master/docs/GettingStarted.md#environment-variables\n# DD_TRACE_AGENT_URL=\n\n\n\n## IP look up configuration\n## ref https://github.com/alexreisner/geocoder/blob/master/README_API_GUIDE.md\n## works only on accounts with ip look up feature enabled\n# IP_LOOKUP_SERVICE=geoip2\n# maxmindb api key to use geoip2 service\n# IP_LOOKUP_API_KEY=\n\n## Rack Attack configuration\n## To prevent and throttle abusive requests\n# ENABLE_RACK_ATTACK=true\n\n\n## Running chatwoot as an API only server\n## setting this value to true will disable the frontend dashboard endpoints\n# CW_API_ONLY_SERVER=false\n\n## Development Only Config\n# if you want to use letter_opener for local emails\n# LETTER_OPENER=true\n# meant to be used in github codespaces\n# WEBPACKER_DEV_SERVER_PUBLIC=\n\n# If you want to use official mobile app,\n# the notifications would be relayed via a Chatwoot server\nENABLE_PUSH_RELAY_SERVER=true\n"
```

MacOS Big Sur 11.6
Docker version 20.10.8, build 3967b7d

**Comment by lybrus** [comment]
With value **accounts@chatwoot.com** is no errors

**Comment by lybrus** [comment]
Yes, it works. It would be nice if you fix it in the template.

**Comment by lybrus** [comment]
Fixed #3226
